src/__tests__/operate.test.js

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Calculator from '../components/Calculator';
+
+describe('Testing operators', () => {
+  test('See if passing the multiply operator works properly', async () => {
+    render(<Calculator />);
+    await userEvent.click(screen.getByRole('button', { name: /2/ }));
+    await userEvent.click(screen.getByRole('button', { name: 'x' }));
+    await userEvent.click(screen.getByRole('button', { name: /7/ }));
+    await userEvent.click(screen.getByRole('button', { name: /=/ }));
+
+    const el = screen.getByTestId('testp').innerHTML;
+    expect(el).toBe('14');
+    expect(el).not.toBe(null);
+  });
```

Comment:
- [ ] Great job. Please kindly test the other operation methods of `addition,` `division,` and `modulo`

Reply:
Thank you for the observation! I just applied the suggested changes. 
![PrayingCatGIF](https://user-images.githubusercontent.com/50700379/201225394-2edea51f-4b13-4a51-abe6-b3af0febfaef.gif)